_includes/tutorials/anomaly-detection/ksql/markup/dev/create-accounts-to-monitor-table.adoc

```diff
@@ -0,0 +1,17 @@
+A single transaction to one of the companies in the suspicious_names table is probably okay, but multiple transactions to one or more of those companies in a 24-hour period is alarming. ksqlDB gives us the ability to see if alarming activity is/was present for a particular user with the following query.
+
+[source,sql]
+----
+CREATE TABLE accounts_to_monitor
+    WITH (kafka_topic='accounts_to_monitor', partitions=1, value_format='JSON') AS
+    SELECT TIMESTAMPTOSTRING(WINDOWSTART, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_START, // <1>
+           TIMESTAMPTOSTRING(WINDOWEND, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_END,
+           USERNAME
+    FROM suspicious_transactions
+    WINDOW TUMBLING (SIZE 24 HOURS) // <2>
+    GROUP BY USERNAME
+    HAVING COUNT(*) > 3; // <3>
+----
+<1> The timestamps here are important, they tell us what interval of time suspicious activity occurred.
+<2> The `WINDOW TUMBLING` part of the query allows us to do an aggregation with distinct time boundaries. In this case our window is fixed at a length of 24 hours, does not allow gaps, and does not allow overlapping. Other types of windows are explained in the "Collect data over time" section of Kafka-Tutorials. If further explanation is need, checkout the link:https://docs.ksqldb.io/en/latest/concepts/time-and-windows-in-ksqldb-queries/#windows-in-sql-queries[*ksqlDB documentation about windows*]. It contains in depth descriptions and visualizations.
```

Comment:
This is a lot of information that is essentially buried in a "footnote".  Readers often skip this as a result.  Can you move most of this out of the footnote and into the text area?

Reply:
This was a tricky request... The numbers in the code + "footnote" are called callouts in asciidoc and are meant for annotations. I did find a way to embed the callout icons in paragraphs... hopefully that is more apparent?